Bee balance

Pdiddy973 opened the pull request:
bee balance still working on it...

- added/updated a few ore textures
- remove compound bees
- renamed crystal bee to quartz bee
- added fluorite bee
- updated lang file
~~
- still to re-balance from ingots to ores, and add breed chance

cbredlow commented:
Don't forget to make the change to draconic bees, as the parent is refined_obsidian: https://github.com/AllTheMods/ATM-6/blob/32e958b5e23ba2c50e53262654ecb4b4c9550b91/config/resourcefulbees/bees/bees_mob/bred/Dragonic.json#L25

cbredlow reviewed (changes requested):
Line 25 of draconic bees has a parent as refined_obsidian, and may cause issues if it's not updated.

Pdiddy973 commented:
updated bee jsons...
changed all the ore bee outputs to ore, with a 50% chance and 1 % chance for 2 ingots. ran all of them through beautifier.io.

changed several bee "flowers" to help with bees all going to one type

added breed chance, starting at 80% for common bees down to 60% for modium

updated feed amounts form ingots/nuggets to blocks

updated time in hive to start at 1200 ticks, up to 4800 for modium

made sure all breedable bees had proper parents
